client/app/pages/ItemsList.jsx

```diff
@@ -0,0 +1,320 @@
+import { isString, isFunction, bind, map, extend, omit } from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Table from 'antd/lib/table';
+import Input from 'antd/lib/input';
+import Select from 'antd/lib/select';
+import { Paginator } from '@/components/Paginator';
+import { BigMessage } from '@/components/BigMessage';
+import { PageHeader } from '@/components/PageHeader';
+import { TagsList } from '@/components/TagsList';
+import { LivePaginator } from '@/lib/pagination';
+import { $location, $route } from '@/services/ng';
+import { clientConfig } from '@/services/auth';
+
+export default class ItemsList extends React.Component {
+  static sidebarMenu = [];
+
+  static listColumns = [];
+
+  static orderSeparator = '-';
+
+  static defaultOrder = '-created_at';
```

Comment:
FYI, there is a popular convention of defining such properties as "configuration" vars outside the class.

```js
import { $location, $route } from '@/services/ng';
import { clientConfig } from '@/services/auth';

const ORDER_SEPARATOR = '_';
const DEFAULT_ORDER = '-created_at';

export default class ItemsList extends React.Component {
  ...
}
```

It also allows to easily export certain vars from other locations that may need it.
```js
import { DEFAULT_ORDER } from '@/pages/ItemsList';
```